packages/core/src/login/webview/vue/amazonq/backend_amazonq.ts

```diff
@@ -36,15 +35,13 @@ export class AmazonQLoginWebview extends CommonAuthWebview {
     }
 
     async fetchConnections(): Promise<AwsConnection[] | undefined> {
-        if (!isExtensionInstalled(VSCODE_EXTENSION_ID.awstoolkit)) {
+        const toolkitApi = getToolkitApi()
+        if (!toolkitApi) {
             return undefined
         }
-        await activateExtension(VSCODE_EXTENSION_ID.awstoolkit)
```

Comment:
Could add this back, but it's kind of pointless because Toolkit activates on `onStartupFinished`.